server/routes/chat.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+const { protect } = require('../middleware/auth');
+const {
+  getChat,
+} = require('../controllers/chat');
+
+router.route('/:user1/:user2/:limit?').get(protect, getChat);
```

Comment:
Not sure how you'd want to use this route, though. Maybe you can have a second look at it or explain better :sweat_smile:

Reply:
The chats will have a many-to-many relationship so we need user1 **and** user2 to locate their unique chat. I added this route for now because I don't know how the chat logic will be implemented on the client. Once the chat logic is implemented on the client I think it will be easier to give this route a better name. 
Do you have any suggestions for naming this route? Maybe prefix it with 'chat'?